templates/track.php

```diff
@@ -0,0 +1,137 @@
+<?php get_header(); ?>
+<div ng-app="shopping-cart-demo">
+  <div ng-controller="CheckoutController">
+    <div class="bnomics-order-container">
+      <!-- Heading row -->
+      <div class="bnomics-order-heading">
+        <div class="bnomics-order-heading-wrapper">
+          <div class="bnomics-order-id">
+            <span class="bnomics-order-number" ng-cloak><?= __('Order #', 'blockonomics-bitcoin-payments') ?>{{order.order_id}}</span>
+          </div>
+        </div>
+      </div>
+      <!-- Amount row -->
+      <div class="bnomics-order-panel">
+        <div class="bnomics-order-info" ng-init="altcoin_waiting=true">
+          <div class="bnomics-bitcoin-pane" ng-hide="show_altcoin != 0" ng-init="show_altcoin=1"></div>
+          <div class="bnomics-altcoin-pane" ng-style="{'border-left': (altcoin_waiting)?'none':''}" ng-hide="show_altcoin != 1">
+            <div class="bnomics-altcoin-waiting" ng-show="altcoin_waiting" ng-init="altcoin_waiting=true" ng-cloak>
+              <!-- WAITING_FOR_DEPOSIT -->
+              <div class="bnomics-btc-info" style="display: flex;flex-wrap: wrap;" ng-show="order.altstatus == 'waiting'" ng-cloak>
```

Comment:
Unnecessary code ... qr code would never shown on altcoin tracking page

Reply:
Currently, all the altcoin tracking happens on the tracking page including waiting for deposit.